libp2pdht/private/eth/p2p/discoveryv5/protocol.nim

```diff
@@ -1032,6 +1032,76 @@ func init*(
     bitsPerHop: bitsPerHop
   )
 
+proc newProtocol*(
```

Comment:
@elcritch this is the removed code from master.

Reply:
Ah yes, I'll see if I can merge it with @yyoncho's changes